projects/examples/src/components/datagrid/datagrid-row-select.example.component.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import { GridDataFetchResult, GridState, GridColumn, GridSelectionType } from '@vcd/ui-components';
+
+interface Data {
+    href: string;
+}
+
+/**
+ * A component that holds an example of the grid selection capability.
```

Comment:
Please add further details and avoid repetitive words like "A component that holds". For example:

> Logs the selected row to the console when the selection changes.

It doesn't need to be exhaustive but it should try to say more than what I can tell from the names in the code: example component, row selection